scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -94,13 +94,20 @@
             Executors.newFixedThreadPool(getFileTransferThreadPoolSize(),
                     new NamedThreadFactory("FileTransferThreadPool"));
 
+    /**
+     * Mainly for testing purposes
+     */
+    protected TaskProActiveDataspaces() {
+
+    }
+
```

Comment:
If the aim of the visibility is to test then the visibility "package" is more accurate than the visibility "protected".